tools/wptserve/wptserve/response.py

```diff
@@ -203,10 +203,12 @@ def iter_content(self, read_file=False):
         elif isinstance(self.content, str):
             yield self.content.encode(self.encoding)
         elif hasattr(self.content, "read"):
-            if read_file:
-                yield self.content.read()
-            else:
-                yield self.content
+            while True:
+                read = self.content.read(64 * 1024)
```

Comment:
I think this deserves a comment. Why are we manually chunking into 64kB chunks when reading?

Reply:
This was meant to match the behaviour of `ResponseWriter.write_content_file`, except that seems to be 32KiB chunks. Why? Presumably to just avoid ending up with potentially vast files in memory.

Reply:
OK, can you add that as a comment? :)

Reply:
Done!